src/palace.pyx

```diff
@@ -1701,7 +1725,133 @@ cdef class AuxiliaryEffectSlot:
+            return NotImplemented
+        other_effect: Effect = other
+        return self.impl >= other_effect.impl
+
+    def __bool__(self) -> bool:
+        return <boolean> self.impl
+
+    @setter
+    def reverb_properties(self, value: dict) -> None:
+        """The effect with the specified reverb properties.
+        It will automatically downgrade to the Standard Reverb effect
+        if EAXReverb effect is not supported.
+        """
+        cdef EFXEAXREVERBPROPERTIES properties
+        try:
+            properties.flDensity = value['density']
+            properties.flDiffusion = value['diffusion']
+            properties.flGain = value['gain']
+            properties.flGainHF = value['gain_hf']
+            properties.flGainLF = value['gain_lf']
+            properties.flDecayTime = value['decay_time']
+            properties.flDecayHFRatio = value['decay_hf_ratio']
+            properties.flDecayLFRatio = value['decay_lf_ratio']
+            properties.flReflectionsGain = value['reflections_gain']
+            properties.flReflectionsDelay = value['reflections_delay']
+            properties.flReflectionsPan[0] = value['reflections_pan'][0]
+            properties.flReflectionsPan[1] = value['reflections_pan'][1]
+            properties.flReflectionsPan[2] = value['reflections_pan'][2]
+            properties.flLateReverbGain = value['late_reverb_gain']
+            properties.flLateReverbDelay = value['late_reverb_delay']
+            properties.flLateReverbPan[0] = value['late_reverb_pan'][0]
+            properties.flLateReverbPan[1] = value['late_reverb_pan'][1]
+            properties.flLateReverbPan[2] = value['late_reverb_pan'][2]
+            properties.flEchoTime = value['echo_time']
+            properties.flEchoDepth = value['echo_depth']
+            properties.flModulationTime = value['modulation_time']
+            properties.flModulationDepth = value['modulation_depth']
+            properties.flAirAbsorptionGainHF = value['air_absorption_gain_hf']
+            properties.flHFReference = value['hf_reference']
+            properties.flLFReference = value['lf_reference']
+            properties.flRoomRolloffFactor = value['room_rolloff_factor']
+            properties.iDecayHFLimit = value['decay_hf_limit']
+        except KeyError:
+            pass
+        self.impl.set_reverb_properties(properties)
+        
+    @setter
+    def chorus_properties(self, value: dict) -> None:
+        """The effect with the specified chorus properties.
+        It will be thrown if EAXReverb effect is not supported.
+        """
+        cdef EFXCHORUSPROPERTIES properties
+        try:
+            properties.iWaveform = value['waveform']
+            properties.iPhase = value['phase']
+            properties.flRate = value['rate']
+            properties.flDepth = value['depth']
+            properties.flFeedback = value['feedback']
+            properties.flDelay = value['delay']
+        except KeyError:
+            pass
```

Comment:
This is for quick debugging.  Drop the `try`.  The same for the other setter.

Reply:
Incorrect indentation.

Reply:
drop `try` only?

Reply:
The requested change is done, but the changed code is misindented (see Travis).

As a rule of thumb, before every commit you have to:
* Test style (flake8)
* Test build (pip3 install .)
* Test correctness (pytest)
* Review changes (git status and git diff)

The tests can be run altogether by tox but it'd be a bit slower (since it need to setup a venv for you and build with trace on).

Reply:
Can you clarify this first line? i.e 
"The requested change is done, but the changed code is misindented"

Reply:
If you remove `try` you'll have to reindent the block.

https://github.com/McSinyx/palace/blob/new-eff/src/palace.pyx#L1814-1821

should be

```python
        cdef EFXCHORUSPROPERTIES properties
        properties.iWaveform = value['waveform']
        properties.iPhase = value['phase']
        properties.flRate = value['rate']
        properties.flDepth = value['depth']
        properties.flFeedback = value['feedback']
        properties.flDelay = value['delay']
        self.impl.set_chorus_properties(properties)
```

Reply:
ah! I though it must be a tab there, thanks!